test/parallel/test-stream-readable-resumeScheduled.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { Readable, Writable } = require('stream');
+
+const w = new Writable();
+w._write = (chunk, encoding, cb) => {};
+
+// pipe() test case
+const r = new Readable({ read() {} });
+// event 'data' test case
+const r2 = new Readable({ read() {} });
```

Comment:
Can you separate these using block scopes, as done in a lot of other tests. Then, you can reuse the same variable names.